SwiftWisdomTests/StandardLibrary/String/String+IndexingTests.swift

```diff
@@ -2,6 +2,30 @@ import XCTest
 
 class String_IndexingTests: XCTestCase {
     
+    func testRangeSubscriptReturnsProperString() {
+        XCTAssertEqual("hello"[0..<0], "")
+        XCTAssertEqual("hello"[0..<1], "h")
+        XCTAssertEqual("hello"[0..<5], "hello")
```

Comment:
I have seen the 'safe behaviour' implemented in swift internal apis with 'safe:' as the parameter name. Some of our own extensions use this, like Data extensions. I would vote for keeping it this way, with OOB crashing, and leave it up to the caller to manage.